BoxContentSDK/BoxContentSDK/Requests/BOXCollectionFavoritesRequest.h

```diff
@@ -10,4 +10,7 @@
 //Perform API request and any cache update only if refreshBlock is not nil
 - (void)performRequestWithCompletion:(BOXCollectionBlock)completionBlock;
 
+- (void)performRequestWithCached:(BOXCollectionBlock)cacheBlock
```

Comment:
Different issue but I've spoken with a few people about how "cached:" and "refreshed:" isn't quite right for the names here.
Not blocking for this review.